src/main/java/Duke.java

```diff
@@ -29,42 +28,80 @@ public static void Greet(){
     }
 
     public static void Echo(){
+        boolean continueChat;
```

Comment:
I like how all the boolean variables are named such that I can easily tell its purpose as a boolean/judgement.